src/storage.js

```diff
@@ -23,4 +25,39 @@ module.exports = {
   getFeeds: () => redis.smembers(FEEDS),
 
   getFeed: feedID => redis.hgetall(feedID),
+
+  addPost: async post => {
+    await redis
+      .multi()
+      .hmset(
+        // using guid as keys as it is unique to posts
+        post.guid,
+        'author',
+        post.author,
+        'title',
+        post.title,
+        'link',
+        post.link,
+        'content',
+        post.content,
+        'text',
+        post.text,
+        'updated',
+        post.updated,
+        'published',
+        post.published,
+        'url',
+        post.url,
+        'site',
+        post.site
+      )
+      .zadd(POSTS, new Date(post.published).getTime(), post.guid) // sort set by published date as scores
```

Comment:
I'm still worried about `post.published` here.  I think it should already be a `Date`, and you should just call `getTime()` on it.  cc @jatinAroraGit, who is working on the `Post` class.